test/commands/share.js

```diff
@@ -0,0 +1,96 @@
+import sinon from 'sinon';
+
+import * as addAll from '../../src/scripts/add-all';
+import * as add from '../../src/scripts/add';
+import * as bump from '../../src/scripts/bump';
+import * as compare from '../../src/scripts/compare';
+import * as create from '../../src/scripts/create';
+import * as freeze from '../../src/scripts/freeze';
+import * as initLib from '../../src/scripts/init-lib';
+import * as init from '../../src/scripts/init';
+import * as link from '../../src/scripts/link';
+import * as pull from '../../src/scripts/pull';
+import * as push from '../../src/scripts/push';
+import * as remove from '../../src/scripts/remove';
+import * as session from '../../src/scripts/session';
+import * as status from '../../src/scripts/status';
+import * as update from '../../src/scripts/update';
+
+import * as runWithTruffle from '../../src/utils/runWithTruffle';
+import Session from '../../src/models/network/Session';
+import program from '../../src/bin/program';
+
+program.Command.prototype.parseReset = function() {
```

Comment:
I'm thinking if, instead of resetting the program, we could have a function to create a new one a reconfigure it on every test, so we don't need to depend on commander's internal structure. On the other hand, this approach works, is more concise, and also faster, so I guess it's the best option.